crates/cli/src/args/encrypt.rs

```diff
@@ -28,23 +28,23 @@ impl ConfigArg for EncryptArgs {
 
 impl MergeConfig for EncryptArgs {
     fn merge_config(&mut self, config: Config) {
+        let file_path = self.input_args.file.as_deref().unwrap_or(&Path::new(""));
+        let file_path = file_path.to_string_lossy();
```

Comment:
Getting a bit nitpicky, sorry 🙈 but maybe call this `file_path_str`. I dislike reusing variable names when the previous value has been transformed to something else. (similar to saying `x = f(x)`)

Reply:
I did it that way because this is exactly what the `refresh` code path does, in order to be consistent...

Reply:
Ahh, fair. My bad.